pyproject.toml

```diff
@@ -104,3 +104,21 @@ exclude = [
 [[tool.mypy.overrides]]
 module = "pyperf"
 ignore_missing_imports = true
+
+[tool.ruff]
```

Comment:
Please split to `.ruff.toml` and also exclude all benchmark directories from linting.